lib/publisher.js

```diff
@@ -17,37 +17,41 @@
 const rclnodejs = require('bindings')('rclnodejs');
 const debug = require('debug')('rcl_publisher');
 
+/** Class representing a Publisher in ROS */
 class Publisher {
   /**
-   * Represents a ROS Publisher
-   * @constructor
-   * @param {RclHandle} publisherHandle - A RclHandle object of publisher.
-   * @param {RclHandle} nodeHandle - A RclHandle object of node.
-   * @param {Object} typeClass - The class of the topic message.
+   * Create a publisher
+   * @param {RclHandle} handle - A RclHandle object of publisher.
+   * @param {RclHandle} nodeHandle - A RclHandle object of node, to which the publisher belongs.
+   * @param {function} typeClass - The class representing the message.
    * @param {string} topic - The name of the topic.
    */
-  constructor(publisherHandle, nodeHandle, typeClass, topic) {
-    this._handle = publisherHandle;
+  constructor(handle, nodeHandle, typeClass, topic) {
```

Comment:
Is this constructor invisible to users? I'm assuming `node.createPublisher()` is the only interface we expect users to know